src/tank_risk.py

```diff
@@ -453,24 +464,24 @@ def start(self, tankPnts, mapDocument):
             if featureName not in TankResult.attributesForFeature:#The riskFeatureFactory should not receive unkown layers.
                 arcpy.AddWarning("Unkown risk layer: {}".format(self.parseName(featureName)))
                 continue
-                    
+
             arcpy.AddMessage("Processing: {}".format(featureName))
             rf = self.riskFeatureFactory(riskFeature)
             self.riskFeatureNameOrder.append(rf.layerName)#Keep track of name order for output field ordering.
-                
+
             rf.createNearTable(tankPnts)
             resultTime = time.time()
             rf.updateTankResults()
             arcpy.AddMessage("  -Completed: {} Time: {:.2f} sec".format(featureName, time.time() - rfStartTime))
             print "results: {}".format(time.time() - resultTime)
-            
+
         resultRows = TankResult.getOutputRows(self.riskFeatureNameOrder)
         self.createOutputTable(resultRows)
-        
+
 
 if __name__ == '__main__':
-    
-    version = "0.7.1"
+
+    version = "1.0.1"
```

Comment:
when you bump minor, patch resets. When you bump major, minor and patch reset. This should be 1.0.0 not that it matters much.